census/estimator/run_preprocessing.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Runs the preprocessing job to produce records for training."""
+
+import argparse
+import ConfigParser
+import logging
+import os
+import sys
+
+import apache_beam as beam
+
+from preprocessing import preprocess
+
+
+def _parse_arguments(argv):
+  """Parses command line arguments."""
+  parser = argparse.ArgumentParser(
+      description='Runs preprocessing on census train data.')
+  parser.add_argument(
+      '--project_name',
```

Comment:
Minor nit: consider using `project_id` as it seems to be the common term used to refer to the Cloud project in documentation.